### PR TITLE
Added a sleep method.

### PR DESCRIPTION
**Pull request by ernestmc**
This addresses issue #2.
Adds a sleep method to sleep for milliseconds but internally updating the robot state like the speed ramps.

### CODE REVIEW

File: library.properties [hunk 1]
```diff
@@ -4,6 +4,6 @@ author=Lucas Chiesa <lucas.chiesa@gmail.com>
 maintainer=Ernesto Corbellini <ecorbe@gmail.com>
 sentence=A library that makes using your Dreamster a breeze.
 paragraph=This library supports all the basic functions of the Dreamster robot
-category=Robotics
+category=Device Control
```
Comment (tulku) [inline library.properties:7]: why this change?

Comment (ernestmc) [reply library.properties:7]: Because it was giving an error when loading the library. According to the [specification](https://github.com/arduino/Arduino/wiki/Arduino-IDE-1.5:-Library-specification) it must be one of the listed values.
